.byebug_history

```diff
@@ -0,0 +1,34 @@
+continue
```

Comment:
This is why you should never do `git add .`. You never know what you're committing blindly. This should be added to the `.gitignore`.